test/CMakeLists.txt

```diff
@@ -552,7 +552,7 @@ if(LINUX
 
     if(EXAMPLES AND NOT UMF_DISABLE_HWLOC)
         add_test(
-            NAME umf_standalone_examples
+            NAME umf-standalone_examples
```

Comment:
out of curiosity - does it make any difference? or you just want to unify with other binaries? I'm asking, because to be precise our tests have prefix `umf_test-`

Reply:
Just to unify:
![image](https://github.com/user-attachments/assets/d917c32b-8569-4cf1-803e-e2fc57cee092)